src/MSALWrapper/PCACache.cs

```diff
@@ -27,6 +28,10 @@ internal class PCACache
         private static KeyValuePair<string, string> linuxKeyRingAttr1 = new KeyValuePair<string, string>("Version", "1");
         private static KeyValuePair<string, string> linuxKeyRingAttr2 = new KeyValuePair<string, string>("ProductGroup", "Microsoft Develoepr Tools");
 
+        // Plain text cache fallback for headless Linux
+        private const string PlainTextCacheDir = ".azureauth";
+        private const string PlainTextCacheFileName = "msal_cache.json";

```

Comment:
Do you foresee any issues with the fact that we use `msal_{tenantId}.cache` for the normal cache filename, but just `msal_cache.json` for the plaintext fallback? Do we not need to worry about different tenants in this way for the plaintext case?